schema.sql

```diff
@@ -6,6 +6,6 @@ CREATE TABLE animals (
     date_of_birth DATE,
     escape_attempts INTEGER,
     neutered BOOLEAN,
-    weight_kg NUMERIC
+    weight_kg NUMERIC,
+    species character varying
```

Comment:
Required
- Please use the correct SQL statement for adding a new column to existing table. See code example
`ALTER TABLE animals ADD species varchar(255)`